module-bsp/board/rt1051/bsp/eink/ED028TC1.cpp

```diff
@@ -138,7 +138,7 @@ static uint8_t s_einkMaskLut_2Bpp[16] = {0, 0, 0, 0, 1, 1, 1, 1, 2, 2, 2, 2, 3,
  *
  * @return
  */
-static uint8_t *s_EinkTransformFrameCoordinateSystem_1Bpp(uint8_t *dataIn,
+static uint8_t *s_EinkTransformFrameCoordinateSystem_1Bpp(const uint8_t *dataIn,
```

Comment:
consider moving all of those uint's into `std::uint` ;)

Reply:
There are many `uint`s in this file so I'd prefer to not do this on this occasion since it's outside of the scope of this PR. ;)